### PR TITLE
Media message

### PR DESCRIPTION
**Pull request by dawinValenzuela**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [x] New feature (non-breaking change which adds functionality)

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**Storybook Link** (Link to a local storybook url to review new functionality)
https://storybook-media-message-7v6rokdl5q-uc.a.run.app/?path=/story/composer-primitive-chat-conversation-mediamessage--basic

### CODE REVIEW

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { MediaMessageBorderColor, MessagePlacement } from './MediaMessage'
+
+export const getStyles = (
+  placement: MessagePlacement,
+  borderColor: MediaMessageBorderColor,
+  hasTime: boolean
+) => {
+  const { wrapperClasses, messageClasses } = getMessagePlacementClasses(placement, borderColor, hasTime);
+  const timeClasses = hasTime && 'mt-1 mb-2';
```
Comment (guilean) [inline packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.styles.tsx:9]: Why do you need to give margin top to time and margin bottom to baseWrapper when hasTime? Could you just use a margin top on time whenever is time and avoid giving margin top to baseWrapper?

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    const image = screen.getByAltText('title');
+    expect(image.getAttribute('src')).toBe('url');
+  });
+  it('should render primary 25 border color', () => {
+    render(<MediaMessage
+      image={fakeImage}
+      borderColor={MediaMessageBorderColor.Primary25}
+    />);
+    const imageWrapper = screen.getByRole('button');
+    expect(imageWrapper.classList).toContain(MediaMessageBorderColor.Primary25);
```
Comment (guilean) [inline packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:26]:
```suggestion
    expect(imageWrapper).toHaveClass(MediaMessageBorderColor.Primary25);
```

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  });
+  it('should render secondary 25 border color', () => {
+    render(<MediaMessage
+      image={fakeImage}
+      borderColor={MediaMessageBorderColor.Secondary25}
+    />);
+    const imageWrapper = screen.getByRole('button');
+    expect(imageWrapper.classList).toContain(MediaMessageBorderColor.Secondary25);
```
Comment (guilean) [inline packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:34]:
```suggestion
    expect(imageWrapper).toHaveClass(MediaMessageBorderColor.Secondary25);
```

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  });
+  it('should render black 10 border color', () => {
+    render(<MediaMessage
+      image={fakeImage}
+      borderColor={MediaMessageBorderColor.Black10}
+    />);
+    const imageWrapper = screen.getByRole('button');
+    expect(imageWrapper.classList).toContain(MediaMessageBorderColor.Black10);
```
Comment (guilean) [inline packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:42]:
```suggestion
    expect(imageWrapper).toHaveClass(MediaMessageBorderColor.Black10);
```

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  });
+  it('should render left placement styles', () => {
+    render(<MediaMessage
+      image={fakeImage}
+      placement={MessagePlacement.Left}
+    />);
+    const wrapper = screen.getByTestId(WRAPPER_MEDIA_MESSAGE_DEFAULT_TEST_ID);
+    expect(wrapper.classList).toContain('items-start');
```
Comment (guilean) [inline packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:50]:
```suggestion
    expect(wrapper).toHaveClass('items-start');
```

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  });
+  it('should render right placement styles', () => {
+    render(<MediaMessage
+      image={fakeImage}
+      placement={MessagePlacement.Right}
+    />);
+    const wrapper = screen.getByTestId(WRAPPER_MEDIA_MESSAGE_DEFAULT_TEST_ID);
+    expect(wrapper.classList).toContain('items-end');
```
Comment (guilean) [inline packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:58]:
```suggestion
    expect(wrapper).toHaveClass('items-end');
```

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  });
+  it('should render time with styles', () => {
+    render(<MediaMessage
+      image={fakeImage}
+      time="time"
+    />);
+    const time = screen.getByText('time');
+    expect(time.parentElement).toHaveClass('mt-1 mb-2');
```
Comment (guilean) [inline packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:66]: Fix once you change the margin commented above

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.tsx [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+type Props = {
+  className?: string;
+  borderColor?: MediaMessageBorderColor;
+  time?: string;
+  placement?: MessagePlacement;
+  image: ImageType;
+  onClick?: () => void;
+  testId?: string;
+}
```
Comment (guilean) [inline packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.tsx:33]: Yesterday we mentioned that this component should be capable to render media + regular message (?) Am I right? Or do we want to create a wrapper on top of this component? @tkanzakic

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/stories/MediaMessage.stories.tsx [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+export const basic = () => (
+  <div className="p-10">
+    <MediaMessage
+      time={text("Time", "15:05 PM")}
+      borderColor={select('Border Color', MediaMessageBorderColor, MediaMessageBorderColor.Secondary25)}
+      placement={select('Placement', MessagePlacement, MessagePlacement.Left)}
+      image={
+        {
+          title: 'title',
+          url: 'https://images.ctfassets.net/o9153kt66j4s/2ClCasVvBEEKCAnWbctHQt/195c416b76203751ad8d62f231e84855/Screen_Shot_2020-07-30_at_2.07.47_PM.png',
```
Comment (guilean) [inline packages/components/src/components/primitives/Chat/Conversation/MediaMessage/stories/MediaMessage.stories.tsx:20]: Add knobs

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/stories/MediaMessage.stories.tsx [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+export const withMultipleMessages = () => (
+  <div className="p-10">
+    <MediaMessage
+      time={text("Time", "15:05 PM")}
+      borderColor={select('Border Color', MediaMessageBorderColor, MediaMessageBorderColor.Secondary25)}
+      placement={select('Placement', MessagePlacement, MessagePlacement.Left)}
+      image={
+        {
+          title: 'title',
+          url: 'https://images.ctfassets.net/o9153kt66j4s/2ClCasVvBEEKCAnWbctHQt/195c416b76203751ad8d62f231e84855/Screen_Shot_2020-07-30_at_2.07.47_PM.png',
```
Comment (guilean) [inline packages/components/src/components/primitives/Chat/Conversation/MediaMessage/stories/MediaMessage.stories.tsx:41]: add knobs

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx (hunk 2 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:26]: done!

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx (hunk 3 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:34]: done!

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx (hunk 4 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:42]: done!

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx (hunk 5 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:50]: done!

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx (hunk 6 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:58]: done!

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/stories/MediaMessage.stories.tsx (hunk 9 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Chat/Conversation/MediaMessage/stories/MediaMessage.stories.tsx:20]: done!

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/stories/MediaMessage.stories.tsx (hunk 10 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Chat/Conversation/MediaMessage/stories/MediaMessage.stories.tsx:41]: done!

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.styles.tsx (hunk 1 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.styles.tsx:9]: done!

File: packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx (hunk 7 above)
Comment (dawinValenzuela) [reply packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.test.tsx:66]: done!

**Pull request closed without merging**